### ISSUE TITLE
Custom icons not displayed correctly in the YAWl Editor 2.3

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#469]
```
What steps will reproduce the problem?
1. Create a 24x24 pixel png and move to icon directory for editor
2. Start Editor and create new specification
3. Icon is displayed correctly in the tollbar on the left but a flaming 
"x-icon" is displayed in the specification.

What is the expected output? What do you see instead?
I would expect the same icon, taht is displayed in the toolbar on the left.

What version of the product are you using? On what operating system?
YAWL Editor 2.3 on Windows 7.


```

Original issue reported on code.google.com by `MarkusRo...@googlemail.com` on 30 Aug 2012 at 7:57

Attachments:
* [Editor Error.png](https://storage.googleapis.com/google-code-attachments/yawl/issue-469/comment-0/Editor Error.png)

**Comment by yawlfoundation** [comment]
```
It also happens in version 2.1.
A few facts that may help:

My language is Portuguese. The "canonical" path to my installation is:
C:\Users\eb_2\Programas\YAWL4Study-2.1
The "non-canonical" path is 
C:\Usuários\eb_2\Programas\YAWL4Study-2.1

I noticed that the predefined icons are treated in a different way. They are 
deployed in the jar file. I even unjared the editor, put my icons in the same 
folder of the
predefined ones, jared the editor again and executed it. It did not work.

Evandro Baccarin

```

Original comment by `evandro....@gmail.com` on 4 Sep 2012 at 11:58

**Comment by yawlfoundation** [comment]
```
resolved in editor 3.0
```

Original comment by `yawl.mic...@gmail.com` on 9 Dec 2013 at 2:46
* Changed state: **Fixed**
